endian/little.go

```diff
@@ -0,0 +1,8 @@
+//go:build 386 || amd64 || arm || arm64 || mips64le || mipsle || ppc64le || riscv64 || wasm
```

Comment:
can you add `loong64` too? I was trying to rebase https://github.com/tailscale/tailscale/pull/6233 but this PR adds a fourth endianness package that Tailscale depends on (atop Tailscale's, https://github.com/josharian/native (updated in https://github.com/josharian/native/commit/a938fb150d47536d63d6bb2f0c4433091cb8c223)), and u-root/uios (which just got support in https://github.com/u-root/uio/commit/301ac5150d9ede678aeceeb941bc03f1bb845850)

Having four packages we depend on is kinda ridiculous. I'll raise this again with upstream to get something in the standard library.

Reply:
https://github.com/WireGuard/wireguard-go/pull/64/commits/f385d0cea766ae3ac65d73d185bd271687ee7b98